### ISSUE TITLE
Direct gcode dump

### ISSUE THREAD

**Issue by billsey** [jeffeb3/sandify#49]
It seems like a no-brainer, but I believe we should support having Sandify download directly to the Arduino via a USB, Wifi or Bluetooth connection. It should look to the ZenXY device as if something like Repetier-Host were connected and sending data. This would allow us to skip a bunch of steps in getting from the design of a pattern to the actual drawing of that pattern on the table.

**Comment by jeffeb3** [comment]
Yes. This has been the intention from the beginning, when we were selecting a language. I think it would be trivial if the user ran sandify on their machine with node.js. I think it is possible though while still running the code client side, but hosted on github. Someone should figure this out :)
